tests/Etsy/methods.json

```diff
@@ -9,6 +9,16 @@
     "visibility": "public",
```

Comment:
The methods.json in tests was very old, so I updated it from src folder. They should be exactly the same now.